lib/plugins/aws/package/compile/events/cloudWatchEvent/index.js

```diff
@@ -11,9 +11,31 @@ class AwsCompileCloudWatchEventEvents {
       'package:compileEvents': this.compileCloudWatchEventEvents.bind(this),
     };
 
-    // TODO: Complete schema, see https://github.com/serverless/serverless/issues/8026
     this.serverless.configSchemaHandler.defineFunctionEvent('aws', 'cloudwatchEvent', {
       type: 'object',
+      properties: {
+        event: { type: 'object' },
+        input: { type: 'string', maxLength: 8192 },
+        inputPath: { type: 'string', maxLength: 256 },
+        inputTransformer: {
+          type: 'object',
+          properties: {
+            inputPathsMap: { type: 'object' },
+            inputTemplates: { type: 'string', maxLength: 8192 },
```

Comment:
It should be `inputTemplate`.

Also we're relying on same configuration in case of `eventBridge` event. It'll be nice to have both schemas matching.

See: https://github.com/serverless/serverless/blob/6961b629e72aada33ff5a3a12f1a04f686b58329/lib/plugins/aws/package/compile/events/eventBridge/index.js#L17-L64